diff --git a/Project/Assets/Sparta/External/AppsFlyer/AppsFlyerTrackerCallbacks.cs b/Project/Assets/Sparta/External/AppsFlyer/AppsFlyerTrackerCallbacks.cs
index 357c59c..b2752e4 100644
--- a/Project/Assets/Sparta/External/AppsFlyer/AppsFlyerTrackerCallbacks.cs
+++ b/Project/Assets/Sparta/External/AppsFlyer/AppsFlyerTrackerCallbacks.cs
@@ -1,9 +1,12 @@
 ﻿using UnityEngine;
 using System.Collections;
 using UnityEngine.UI;
+using System;

 public class AppsFlyerTrackerCallbacks : MonoBehaviour {

+    public Action<string> OnConversionDataReceived;
+
 	public Text callbacks;

 	// Use this for initialization
@@ -19,6 +22,9 @@ public class AppsFlyerTrackerCallbacks : MonoBehaviour {

 	public void didReceiveConversionData(string conversionData) {
 		printCallback ("AppsFlyerTrackerCallbacks:: got conversion data = " + conversionData);
+        if(OnConversionDataReceived != null) {
+            OnConversionDataReceived(conversionData);
+        }
 	}

 	public void didReceiveConversionDataWithError(string error) {
